hardhat.config.js

```diff
@@ -0,0 +1,33 @@
+require("@nomiclabs/hardhat-etherscan");
+
+module.exports = {
+  defaultNetwork: "hardhat",
+  networks: {
+    kovan: {
+      url:  "https://eth-kovan.alchemyapi.io/v2/H8E5dk64j-odPJlWF6TYnBYKjkZHYjuD"
```

Comment:
@mgnsharon this file should not be committed. Contains API keys

Reply:
Moved the url's and api key to .env and added dotenv to hardhat config